### PR TITLE
fix: update links in header and footer

### PR DESCRIPTION
**Pull request by samjwu**
also adds alpha watermark

### CODE REVIEW

**Comment by saadrahim** [comment]
Can we have a preview link?

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = 'rocm-docs-core'
-version = "0.2.0"
+version = "0.2.1"
```
Comment (saadrahim) [inline pyproject.toml]: is commitzen supposed to update automatically? I don't understand how it work yet.

**Comment by samjwu** [comment]
> Can we have a preview link?

https://rocmdocs.amd.com/projects/alpha/en/rtd/

The static content is obtained from the main branch, so the watermark won't appear until this is merged

File: pyproject.toml (hunk 1 above)
Comment (samjwu) [reply pyproject.toml]: Thought it was a convention. Missed that a GitHub Action was added

Reverted in [28585bd](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/28585bd448f822d07418c5741d37667158199915)

File: docs/.gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 /_build
 /_doxygen
+/_images
+/_static
+/_templates
```
Comment (Maetveis) [inline docs/.gitignore]: Building the docs should no longer generate these, that's why I removed them from the ignore list.
If you just delete them from your working tree then they shouldn't come back.

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@ dist/**/*
 **/__pycache__/**/*
 **/*.egg-info/**/*
 .sass-cache/*
+build
```
Comment (Maetveis) [inline .gitignore]:
```suggestion
/build
```
We don't want it applied to sub folders containing `build`. Btw what creates this folder?

File: src/rocm_docs/rocm_docs_theme/static/images/alpha-watermark.svg [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="394.000000pt" height="77.000000pt" viewBox="0 0 394.000000 77.000000"
+ preserveAspectRatio="xMidYMid meet">
+
+<g transform="translate(0.000000,77.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
```
Comment (Maetveis) [inline src/rocm_docs/rocm_docs_theme/static/images/alpha-watermark.svg]:
```suggestion
fill="#a0a0a0" stroke="none">
```

Tweaking this to mid-gray and adjusting the opacity to 20% makes the watermark also visible on the dark theme.

**Review by Maetveis — CHANGES_REQUESTED** [review]
A few suggestion, overall lgtm.

A small notes on commit prefixes:
- docs is for the projects **own** documentation, i.e. changes that only affect this projects docs most of the changes in this PR should be `feat:`
- IMO reverting should either be in `git revert` style `Revert "<original commit>"` or with the type `revert: ` (I would prefer option 1)

Any change that results in downstream changes should be "feat" or "fix", with breaking changes noted when required, as these are the only prefixes that trigger a new version number.

File: docs/.gitignore (hunk 2 above)
Comment (samjwu) [reply docs/.gitignore]: Removed in [c479e7b](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/c479e7bd6089f110ccc0473ff1fe61439152695f)

File: .gitignore (hunk 3 above)
Comment (samjwu) [reply .gitignore]: Done in [c479e7b](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/c479e7bd6089f110ccc0473ff1fe61439152695f)

This gets built when I install a local version of rocm-docs-core for testing via `pip install <dir>/rocm-docs-core`

File: src/rocm_docs/rocm_docs_theme/static/images/alpha-watermark.svg (hunk 4 above)
Comment (samjwu) [reply src/rocm_docs/rocm_docs_theme/static/images/alpha-watermark.svg]: Looks good. Changes made in [7ccebb9](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/87/commits/7ccebb97e512d4eac98eefcae0cc52d5e38b668d)

**Comment by samjwu** [comment]
> A few suggestion, overall lgtm.
> 
> Some notes on commit prefixes:
> 
>     * docs is for the projects **own** documentation, i.e. changes that only affect this projects docs most of the changes in this PR should be `feat:`
> 
>     * IMO reverting should either be in `git revert` style `Revert "<original commit>"` or with the type `revert: ` (I would prefer option 1)
> 
> 
> Any change that results in downstream changes should be "feat" or "fix", with breaking changes noted when required, as these are the only prefixes that trigger a new version number.

I have rebased and edited the commit messages to match.

**Review by Maetveis — APPROVED** [review]

**Pull request merged**